provider/generic_repository.go

```diff
@@ -55,8 +59,16 @@ func (p *GenericProvider) Gather(user string) ([]GitRepository, error) {
 	if p.options.RepositoryName != "" {
 		name = p.options.RepositoryName
 	} else {
+		path := user
+		if _, err := os.Stat(user); err == nil || !os.IsNotExist(err) {
+			if absPath, err := filepath.Abs(user); err == nil {
+				log.Debugf("`%v` is a local path, will use absolute path `%v`", user, absPath)
```

Comment:
Making a debug message will make te message on among all the "line" display. We could make it a warning, but may be overkill. I leave it up to you 🙂

Reply:
Sorry, one extra remark : we had an option `--repo-name` to specify a repo name in that exact case. Now it can still be used to override the name used. Perhaps we can say it in the debug/warning message so that the user knows how to override the repo name ?  
It may be a bit verbose...

Reply:
This message is only for debugging to see if the absolute path is correct.

I've added a warning log: `Name of the repository unspecified (see --repo-name). '%v' has been inferred from the object.`